osu.Game.Rulesets.Osu/Objects/Drawables/Connections/FollowPointRenderer.cs

```diff
@@ -100,15 +100,25 @@ private void update()
                             Scale = new Vector2(1.5f * currHitObject.Scale),
                         });
 
-                        using (fp.BeginAbsoluteSequence(fadeInTime))
+                        HasFailed.BindValueChanged(v =>
                         {
-                            fp.FadeIn(currHitObject.TimeFadeIn);
-                            fp.ScaleTo(currHitObject.Scale, currHitObject.TimeFadeIn, Easing.Out);
-
-                            fp.MoveTo(pointEndPosition, currHitObject.TimeFadeIn, Easing.Out);
-
-                            fp.Delay(fadeOutTime - fadeInTime).FadeOut(currHitObject.TimeFadeIn);
-                        }
+                            fp.ClearTransforms();
+
+                            if (v.NewValue)
+                                fp.ApplyFailTransforms();
+                            else
+                            {
+                                using (fp.BeginAbsoluteSequence(fadeInTime))
+                                {
+                                    fp.FadeIn(currHitObject.TimeFadeIn);
+                                    fp.ScaleTo(currHitObject.Scale, currHitObject.TimeFadeIn, Easing.Out);
+
+                                    fp.MoveTo(pointEndPosition, currHitObject.TimeFadeIn, Easing.Out);
+
+                                    fp.Delay(fadeOutTime - fadeInTime).FadeOut(currHitObject.TimeFadeIn);
+                                }
+                            }
+                        }, true);
 
                         fp.Expire(true);
```

Comment:
i feel this is going to cause issues

Reply:
I'm not sure how? it first clears all the transformations inside a follow point, and apply the following:
1. if it has failed, apply immediate fail transform (no delayed / absolute sequences).
2. if it hasn't, apply normal transformations at the follow point fade in time.

Reasons why?